p2p/pex/pex_reactor_test.go

```diff
@@ -35,6 +37,73 @@ func TestPEXReactorBasic(t *testing.T) {
 	assert.NotEmpty(t, r.GetChannels())
 }
 
+func TestPEXReactorStopPeerWithOutInitPeer(t *testing.T) {
```

Comment:
can we rename this to something which reflects the test's purpose?

I don't understand what the test is testing right now.